app/javascript/components/supplemental_application/sections/preferences/DefaultPanel.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 export const DefaultPanel = () => {
-  return (<div />)
+  return (<div> This is the default panel. Something might be wrong</div>)
```

Comment:
do we want to keep this in?

Reply:
maybe not. I thought It was a good idea. I'm rolling this back